torchtext/datasets/text_classification.py

```diff
@@ -373,6 +408,122 @@ def AmazonReviewFull(*args, **kwargs):
     return _setup_datasets(*(("AmazonReviewFull",) + args), **kwargs)
 
 
+def _generate_imdb_data(key, extracted_files, vocab, tokenizer, removed_tokens):
+    _data = []
+    for fname in extracted_files:
+        if 'urls' in fname:
+            continue
+        elif key in fname:
+            if 'pos' in fname:
+                txt_iter = iter(tokenizer(row) for row in io.open(fname,
+                                                                  encoding="utf8"))
+                _iter = numericalize_tokens_from_iterator(vocab,
```

Comment:
Please stop prefixing variables that aren't private member variables

Reply:
Fixed